ui/src/pages/organizations/add.tsx

```diff
@@ -0,0 +1,100 @@
+import classNames from 'classnames/bind';
+import { Formik, FormikProps } from 'formik';
+import Head from 'next/head';
+import Link from 'next/link';
+import * as Yup from 'yup';
+
+import FileField from 'src/components/Form/FileField';
+import TextField from 'src/components/Form/TextField';
+import FullPageLayout from 'src/components/FullPageLayout';
+import style from './style.module.scss';
+
+const c = classNames.bind(style);
+
+const AddOrg: React.FC<FormikProps<OrganizationFormValues>> = (props) => {
+  return (
+    <>
+      <Head>
+        <title>Snake Oil Softwware - Organizations</title>
```

Comment:
Typo in Softwware. I am sure it is in other places as well. Can you please do a global search and fix all instances of this?

Reply:
fixed this in in **login-api-integration** branch (where create Layout component)